Xamarin.Forms.Core/PlatformConfiguration/AndroidSpecific/CarouselPage.cs

```diff
@@ -0,0 +1,36 @@
+﻿namespace Xamarin.Forms.PlatformConfiguration.AndroidSpecific
+{
+	using FormsElement = Forms.CarouselPage;
+
+	public static class CarouselPage
+    {
+		public static readonly BindableProperty IsSwipePagingEnabledProperty =
```

Comment:
Reminance from a dirty copy paste from AndroidSpecific.TabbedPage. Which I also could have refactored to MultiPage.